src/serializer/ResidualHeapSerializer.js

```diff
@@ -1134,6 +1180,20 @@ export class ResidualHeapSerializer {
     return context;
   }
 
+  _serializeAdditionalFunction(generator: Generator, postGeneratorCallback: () => void) {
```

Comment:
This function shares code with `serializeGenerator` in `_getContext`. Can you factor out a common helper function?